vita3k/mem/include/mem/allocator.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <cstdint>
+#include <mutex>
+#include <vector>
+
+struct BitmapAllocator {
+    std::vector<std::uint32_t> words;
+    std::size_t max_offset;
+
+protected:
+    int ForceFill(const std::uint32_t offset, const int size, const bool or_mode = false);
```

Comment:
Can you make functions snake cases?

Reply:
I thought most of it is this case

Reply:
Many of our code is in snake case including your ngs code. Let's try to make them snake case when possible.

Reply:
I see. Let's goo